### ISSUE TITLE
Attaching a file with a "®" in the file name crashes client on Linux

### ISSUE THREAD

**Issue by QEDeD** [tutao/tutanota#3007]
- [X] This is not a feature request (existing functionality does not work, **not** missing functionality).
 I will request features on [forum](https://www.reddit.com/r/tutanota/) or via support.
- [X] I've searched and did not find a similar issue.

**Describe the bug**
When writing a mail and attaching a file, a dialog saying "The file could not be loaded" then the client just turns into a white screen - and rather annoyingly - the mail being worked on is lost.

After short testing, it seems the crash only happened when the "®" character was present in the file name. After removing it, the crash did not occur.

**To Reproduce**
1. Write a mail
2. Attach any file with a "®" character in the file name
3. See the error

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Pop!_OS 20.10
 - Gnome
 - Tutatnota Version 3.83.2

**Comment by charlag** [comment]
Hi
Could not reproduce it yet?

![Screenshot from 2021-05-03 09-58-58](https://user-images.githubusercontent.com/3099142/116853365-35ed9800-abf6-11eb-9e4b-9fd2084ad5b4.png)

**Comment by QEDeD** [comment]
Hmm, I tested again by creating a file called test with touch. I then attached that file without issue. I made a copy of the filed and changed it's file name to "test®" and the client crashed.

I'll try rebooting my system and see if the crash is still present later.

**Comment by jowlo** [comment]
@QEDeD could or can you still reproduce this ?
I tried when you first posted and once again and it works fine for me. Maybe you could attach the file that leads to the crash here to rule out us using a wrong filename, i think there are multiple ® chars.

**Comment by QEDeD** [comment]
@jowlo sorry for the delay. I tested again and I see the same behavoir (crash). I've uploaded the file I'm testing with as well as a gif showing the error here: https://drive.google.com/drive/folders/1CKAUrNkDgZ-CZNQQTdANPrHYhkWOpL_7?usp=sharing

**Comment by QEDeD** [comment]
@johnbotris 

Tried to do `flatpak run com.tutanota.Tutanota`, but I all the output I saw was generated right after I opened the app, nothing happened when I tested with the special character file:  
```

[2 preload-host-spawn-strategy] Running: /app/bin/zypak-helper child - /app/lib/tutanota/tutanota-desktop --type=zygote
the monkey has been patched
version:   3.83.2
(node:2) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
config up to date
Gtk-Message: 17:54:06.880: Failed to load module "appmenu-gtk-module"
Gtk-Message: 17:54:06.917: Failed to load module "canberra-gtk-module"
Gtk-Message: 17:54:06.917: Failed to load module "canberra-gtk-module"
```

I also checked the debugging terminal but when I attached the file I got:
![image](https://user-images.githubusercontent.com/16562457/117688811-6d8dae80-b1b9-11eb-9ba4-dee22637ccba.png)

I don't seem to have the directory you describe (in the below I go to ~/.config/ and tab to autocomplete :thinking: 

```
➜  .config pwd 
/home/kristoffer/.config
➜  .config ls t
teamviewer/  trackma/     TUDelft/   
```

**Comment by johnbotris** [comment]
@QEDeD Ah, If you're using flatpak then you should find the logs under `~/.var/app/com.tutanota.Tutanota/config/tutanota-desktop/logs/`

In the Developer tools window, if you switch to the `console` tab, then you will be able to see all the output there. Hopefully it doesn't give up before saying what's wrong.

**Comment by johnbotris** [comment]
@QEDeD Great thanks, that helps localize it, I'll see if I can reproduce it now

**Comment by QEDeD** [comment]
Glad I could help

**Comment by ganthern** [comment]
I can't for the life of me reproduce this on Pop or anywhere else, so I'm going to close for now. We can reopen if there are more reports.

**Comment by QEDeD** [comment]
@ganthern Thanks for the effort, as far as I can tell, the issue is resolved in v 3.98.24 :+1:
